package.json

```diff
@@ -35,12 +35,12 @@
     "@babel/plugin-proposal-optional-chaining": "^7.14.5",
     "@babel/plugin-transform-runtime": "^7.15.0",
     "@babel/preset-env": "^7.15.0",
-    "@babel/preset-react": "^7.13.13",
+    "@babel/preset-react": "^7.14.5",
```

Comment:
This is the buggy version of preset-react, we can't update this. Still need to look into what's causing this. So best to leave it at `"^7.13.13"` for now.

Reply:
Aha, good spot! I'll roll it back

Reply:
Looks like there are still problems. I have a feeling that the lockfile is still pulling in dependencies from 7.14.5.

My suggestion would actually just be to close this PR, and open up a new one with the rest of the updates, minus this dependency.

Reply:
ah I missed something with my fix for node-fetch. I think all fixed with latest push

Reply:
https://github.com/guardian/gateway/pull/903